src/theme/styles/message.theme.json

```diff
@@ -0,0 +1,47 @@
+{
+    "message": {
+        "base": {
+            "color": "{{colors.neutral.790}}",
+            "padding": "1rem",
+            "border-radius": "0.625rem",
+            "border-width": "1px"
+        },
+        "styles": {
+            "success": {
+                "background": "{{colors.success.300}}",
+                "color": "{{colors.success.500}}",
+                "border-color": "{{colors.success.500}}"
+            },
+            "info": {
+                "background": "{{colors.info.300}}",
+                "color": "{{colors.info.500}}",
+                "border-color": "{{colors.info.500}}"
+            },
+            "error": {
+                "background": "{{colors.danger.300}}",
+                "color": "{{colors.danger.500}}",
+                "border-color": "{{colors.danger.500}}"
+            },
+            "warning": {
+                "background": "{{colors.warning.300}}",
+                "color": "{{colors.warning.500}}",
+                "border-color": "{{colors.warning.500}}"
+            },
+            "icon": {
```

Comment:
**issue:** take the icon, title, description, and button keys out of styles.

Though the styles `success, info, error, and warning` are correct as `styles` these keys are not talking about styles but are children of the component. 

I think we should put these under another key. maybe `components`

```
"message": {
  ...
  "components" : {
    "title": {
      ...
    }
  },
  "styles": {
    ..
  }
}
```

you can then change `styles` into `components` in the helper function.